PolicyProject1/src/main/java/com/example/demo/model/request/CreatePolicyRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.example.demo.model.request;
+
+import com.example.demo.model.PaymentTerm;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
```

Comment:
using this annotation no need to create constructor parametrized or non parametrized.